apps/server/src/modules/lti-tool/lti-tool.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { LtiToolRepo } from '@shared/repo';
+import { LegacyLogger } from '@src/core/logger';
+import { LtiToolService } from './service';
+
+@Module({
+	providers: [LtiToolService, LtiToolRepo, LegacyLogger],
+	exports: [LtiToolService],
+})
+export class LtiToolModule {}
```

Comment:
Should we deprecate this?

Reply:
No, the module itself is not deprecated, since we have no replacement to access LTI tools. Only the concept of LTI Tools is deprecated.